<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(13, 10, 11);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>20</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 253, 237);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>20</width>
     <height>181</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 253, 237);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="role">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 253, 237);</string>
   </property>
   <property name="text">
    <string>you are </string>
   </property>
  </widget>
  <widget class="QLabel" name="turn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 253, 237);</string>
   </property>
   <property name="text">
    <string>Opponent's Turn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mid_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="top_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="top_mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="top_right">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mid_right">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bot_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bot_mid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bot_right">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; 
border: none;
color: rgb(255, 253, 237);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>top_left</tabstop>
  <tabstop>top_mid</tabstop>
  <tabstop>top_right</tabstop>
  <tabstop>mid_left</tabstop>
  <tabstop>mid</tabstop>
  <tabstop>mid_right</tabstop>
  <tabstop>bot_left</tabstop>
  <tabstop>bot_mid</tabstop>
  <tabstop>bot_right</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
